pkg/make/builder.go

```diff
@@ -474,6 +484,41 @@ func bootstrapFromNotary(notaryConfigDir, remoteServerURL, localRepo, gun string
 	return nil
 }
 
+func bootstrapFromTUF(remoteServerURL string, localRepo string) error {
+	req, err := http.NewRequest(http.MethodGet, remoteServerURL+"/repository/root.json", nil)
+	if err != nil {
+		return fmt.Errorf("could not create request to TUF: %w", err)
+	}
```

Comment:
How do we vet this download?

Reply:
Checking the tuf spec and the go-tuf library more, I think we should remove this download, and ship this file as a hardcoded asset instead -- ref [here](https://theupdateframework.github.io/specification/latest/#load-trusted-root). Let me know if that sounds right to you?

Reply:
I think, something like that. The process that our current code _should_ be doing is along the lines of:
1. Check in some initial trust information (this `root.json`, for example)
2. Update a build time copy 
3. Ship the updated version

The intent of (2) is to incur that initial download/sync cost at build time.

Reply:
This is updated now